### PR TITLE
Fix various issues

### PR DESCRIPTION
**Pull request by virgil-serbanuta**

### CODE REVIEW

File: ulm-semantics/main/encoding/encoder.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ module ULM-ENCODING-HELPER
     rule encodeHexBytes(_:String) => .Bytes
         [owise]
     rule encodeHexBytes(S:String)
-        => Int2Bytes(2, String2Base(substrString(S, 0, 2), 16), BE)
+        => Int2Bytes(1, String2Base(substrString(S, 0, 2), 16), BE)
```
Comment (ACassimiro) [inline ulm-semantics/main/encoding/encoder.md:15]: I understand the change in the signature size from 8 Bytes to 2, but what is the motivation for the change here?

Comment (virgil-serbanuta) [reply ulm-semantics/main/encoding/encoder.md:15]: Two hex characters (`substrString(S, 0, 2)`) should become a single byte.

Comment (virgil-serbanuta) [reply ulm-semantics/main/encoding/encoder.md:15]: i.e., "FA" should become b"\xfa" which is one byte

**Review by ACassimiro — APPROVED** [review]

File: ulm-semantics/main/execution/dispatch.md [hunk 2]
```diff
@@ -10,16 +10,18 @@ module ULM-EXECUTION-DISPATCH
...
+                            | setContractOutput(Bytes)
 
     rule
         <k>
             // TODO: For some reason, kompile rejects 'Create' as a variabler name below.
             // Figure out why.
             ulmExecute(Createy:Bool, Pgm:Bytes, _AccountId:Int, Gas:Int)
             => ulmExecute(Createy, struct(ContractTrait, .Map), integerToValue(Gas, u64))
-              ~> #if Createy #then Pgm #else .K #fi
+              ~> #if Createy #then setContractOutput(Pgm) #else .K #fi
```
[the remark this answers is not in the document]
Comment (virgil-serbanuta) [reply ulm-semantics/main/execution/dispatch.md:24]: Yes, Createy should be Create, see the TODO above. However, the code does not compile with that. Also, my current theory is that we have a function called `Create`, and the tokenizer thinks that "Create" is the function-name token, not a variable token, but I didn't test that.
